pkg/cmd/cli/cmd/project.go

```diff
@@ -155,6 +155,7 @@ func (o ProjectOptions) RunProject() error {
 
 	contextInUse := ""
 	namespaceInUse := ""
+	isDefaultContext := false
```

Comment:
The name is generated by us vs named by the user.  I'll change it to 'contextNameIsGenerated'

----- Original Message -----
> > @@ -155,6 +155,7 @@ func (o ProjectOptions) RunProject() error {
> >  
> >  	contextInUse := ""
> >  	namespaceInUse := ""
> > +	isDefaultContext := false
> 
> This is badly named.  I'm trying to figure out what it is.  Is it marking the
> fact that no matching context was found and the name was treated as a
> project name?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2405/files#r30794254